llvm/test/Transforms/LoopVectorize/iv-select-cmp-trunc.ll

```diff
@@ -0,0 +1,285 @@
+; RUN: opt -passes=loop-vectorize -force-vector-interleave=1 -force-vector-width=4 -S < %s | FileCheck %s --check-prefix=CHECK
+; RUN: opt -passes=loop-vectorize -force-vector-interleave=4 -force-vector-width=4 -S < %s | FileCheck %s --check-prefix=CHECK
+; RUN: opt -passes=loop-vectorize -force-vector-interleave=4 -force-vector-width=1 -S < %s | FileCheck %s --check-prefix=CHECK
```

Comment:
Please remove `--check-prefix=CHECK`, as it is the default check prefix.

Reply:
I prefer to keep "--check-prefix=CHECK". Will soon be changed to "--check-prefix=CHECK-VF4IC1 --check-prefix=CHECK".